src/other/Table.js

```diff
@@ -321,11 +321,17 @@
         cells.exit()
             .remove()
         ;
+        if (!this.data().length) {
+            d3.select(".cols-wrapper").remove();        
+        }
+        if (!this.columns().length) {
+            d3.select(".rows-wrapper").remove();
+        }
 
```

Comment:
@GordonSmith I made these conditional so they only execute when the appropriate data is resetting.  I may even be over thinking this.  When the data is emptied and then new data is added, the header and footer re-render fine.  These statements are ONLY necessary when the data is emptied and then the table is rendered.  The fixed header and footer code never runs so they are left in the dom.  Is this even a realistic scenario or am I getting carried away?